src/sentry/api/base.py

```diff
@@ -220,17 +230,8 @@ def paginate(self, request, on_results=None, paginator_cls=Paginator,
         if on_results:
             results = on_results(cursor_result.results)
 
-        headers = {}
-        if cursor_result.hits is not None:
-            headers['X-Hits'] = cursor_result.hits
-        if cursor_result.max_hits is not None:
-            headers['X-Max-Hits'] = cursor_result.max_hits
-        headers['Link'] = ', '.join([
-            self.build_cursor_link(request, 'previous', cursor_result.prev),
-            self.build_cursor_link(request, 'next', cursor_result.next),
-        ])
-
-        return Response(results, headers=headers)
```

Comment:
derp i forgot to add the return statement

will get around to fixing this when im back and have time to see this through